core/dbt/context/providers.py

```diff
@@ -1541,6 +1541,14 @@ def env_var(self, var: str, default: Optional[str] = None) -> str:
         else:
             return super().env_var(var, default)
 
+    @contextproperty()
+    def this(self) -> Optional[str]:
```

Comment:
Elsewhere "this" is a Relation. Here it's just a model name plus the ephemeral_prefix. Might be confusing. Maybe call it something else?

Reply:
I don't think we can we can call it anything else in the provider specifically, because need the `{{ this }}` call in the jinja context to resolve to the CTE name of the fixture we create from `this` provided in the unit test yaml.

Reply:
Ah. Okay :)